statsdaemon.go

```diff
@@ -572,6 +576,24 @@ func tcpListener() {
 	}
 }
 
+func heartbeat() {
+	fileName := "heartbeat.monitor"
```

Comment:
Can you make the argument `--heartbeat-file` and avoid baking in this specific filename?